### DISCUSSION TITLE
请了维多利亚家政来我咖啡馆帮忙，五星好评！！！

### DISCUSSION THREAD

**Discussion by InkVoid-0** [share121/inter-knot#399] [其他]
![0](https://github.com/user-attachments/assets/af08e77a-9d16-452c-aaaf-8e67850b59d3)

就是那位鲨希人小姐脾气好像不怎么好，但服务很周到，希望下次还能请到维多利亚家政！

**Comment by npldqt** [comment]
偷偷告诉你，可以试着给鲨希人投喂一下糖果

**Comment by Bambook-sth** [comment]
我好像在六分街见过这位鲨希人小姐，记得是在录像店门口等人的样子

**Comment by WORLD-executeME** [comment]
我在六分街的录像店里看到过她，似乎和录像店店长很熟的样子，如果想知道和她相处的诀窍说不定可以去找那边的店长问问。

**Comment by NekoWise** [comment]
感谢您的惠顾！欢迎再次叨扰维多利亚家政

**Comment by nightrain2333** [comment]
qwq
